editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -81,8 +82,15 @@ interface SummaryListsBySectionsProps extends SummaryListProps {
 }
 
 function SummaryListsBySections(props: SummaryListsBySectionsProps) {
-  const [hasSections, getSortedBreadcrumbsBySection] = useStore((state) => [
+  const [
+    hasSections,
+    sectionNodes,
```

Comment:
`sectionNodes` and `sectionStatuses()` look unused?

Reply:
correct, good spot - now cleaned up!